R/MAdensity_plot.R

```diff
@@ -15,7 +15,7 @@
 #' model <- ma_continuous_fit(doses, y,
 #'   fit_type = "mcmc", BMR_TYPE = "sd", BMR = 1
 #' )
-#' MAdensity_plot(model)
+#' try(MAdensity_plot(model))
```

Comment:
`--as-cran` now runs all `dontrun` and `donttest` tests so I had to wrap all the potential errors in a `try()` block to make CRAN happy.